src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -1079,6 +1080,27 @@ public void testCreateDatasetRegistration_invalidFileName() {
     assertEquals(HttpStatusCodes.STATUS_CODE_BAD_REQUEST, response.getStatus());
   }
 
+  @Test
+  public void testGetStudy() {
+    Study study = new Study();
+    study.setStudyId(1);
+    study.setName("asdfasdfasdfasdfasdfasdf");
+    when(datasetService.findStudyById(1)).thenReturn(study);
```

Comment:
I think it's also worth adding a test that returns mock datasets so you can ensure every line is covered.